drafts/maui-test_pt1.md

```diff
@@ -0,0 +1,331 @@
+---
+title: How to test a .NET MAUI App Part 1
+domain: software-engineering-corner.zuehlke.com
+tags: dotnet,mvvm,xaml,net-maui,csharp,xunit,testing,moq
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1715274155168/4q4lrfJcY.png?auto=format
+publishAs: NiZarper
+hideFromHashnodeCommunity: false
+saveAsDraft: true
+---
+
+# How to test a .NET MAUI App Part 1
+
+This blog post is the first part of a two-part series on testing a .NET MAUI application. This first post focuses on the service and view model layers, while the second will focus on the UI testing aspect.
+
+## Table Of Contents
+
+- Setup
+- Testing of a service
+- Testing of a view model
+- Takeaways
```

Comment:
**suggestion:** with the `enableToc: true` flag in the frontmatter you'd get a generated TOC linking to the headings (see: https://github.com/Hashnode/Hashnode-source-from-github-template)

Reply:
Thanks for the hint. I added it.